alphafold3_pytorch/inputs.py

```diff
@@ -783,6 +819,7 @@ def alphafold3_input_to_molecule_input(
     # handle molecule ids
 
     molecule_ids = torch.cat(molecule_ids).long()
+    # TODO: do not pad this with zeros anymore, as it will mistakenly treat padded tokens as `ALA`
```

Comment:
oh i see, yea these are for the metal ions and eventual miscellaneous biomolecules

what id do you think it should have? is this that `gap_id` i saw earlier?

Reply:
If you don't know the type of a molecule, I'd say `gap_id=31` would be a safe bet to set these to.